backends/dpdk/dpdkArch.h

```diff
@@ -828,7 +828,7 @@ struct keyInfo {
 class CopyMatchKeysToSingleStruct : public P4::KeySideEffect {
     IR::IndexedVector<IR::Declaration> decls;
     DpdkProgramStructure *structure;
-    bool metaCopyNeeded;
+    bool metaCopyNeeded = false;
```

Comment:
It might be better to add the initialization to `init_apply`. But it seems each of these passes runs only once, so should be OK this way too.

Reply:
Yeah, the logic definitely should be refined here ;)